test/L1/SwapAndBridge.t.sol

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.23;
+
+import { Test, console2, Vm } from "forge-std/Test.sol";
+import { SwapAndBridge } from "src/L1/SwapAndBridge.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { MockERC20LST } from "../../test/mock/MockERC20LST.sol";
+import { MockStandardBridge } from "../../test/mock/MockStandardBridge.sol";
+
+/// @title TestSwapAndBridge
+/// @notice This contract is used to run unit tests for the SwapAndBridge contract.
+contract TestSwapAndBridge is Test {
+    // Address used for unit tests
+    address constant testAccount = address(0xc0ffee);
+
+    // test account balance
+    uint256 testBalance = 1e60;
+
+    function setUp() public {
+        vm.deal(testAccount, testBalance);
+    }
+
+    function test_InvalidConstructor() public {
+        vm.expectRevert("Invalid L1 bridge address.");
+        new SwapAndBridge(address(0), address(0xdead), address(0xbeef));
+
+        vm.expectRevert("Invalid L1 token address.");
+        new SwapAndBridge(address(0xbeef), address(0), address(0xdead));
+
+        vm.expectRevert("Invalid L2 token address.");
+        new SwapAndBridge(address(0xbeef), address(0xdead), address(0));
+    }
+
+    function test_InvalidRecipient() public {
+        SwapAndBridge swapAndBridge = new SwapAndBridge(address(0xdead), address(0xbeef), address(0x1111));
+        vm.expectRevert("Invalid recipient address.");
+        swapAndBridge.swapAndBridgeTo{ value: 1 }(address(0));
+    }
+
+    function test_ReceiveFallback(uint256 tokensPerETH, uint256 value) public {
```

Comment:
```suggestion
    function testFuzz_ReceiveFallback(uint256 tokensPerETH, uint256 value) public {
```

All test functions below which are property based tests need to have `testFuzz` at the beginning.

Reply:
added